kv/memberlist/memberlist_client.go

```diff
@@ -1,14 +1,16 @@
 package memberlist
 
 import (
-	"bytes"
+	bytes "bytes"
```

Comment:
is that intentional? same for `math`.

Reply:
it is not... somehow my `gofmt` decided to rename those packages on my behalf. 🤷‍♂️ 

fixed.